testsuite/domain/src/test/resources/domain-configs/domain-elytron.xml

```diff
@@ -0,0 +1,514 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
No it wasn't thanks. I used the minimal domain config and domain-elytron.xml is now gone.